migrations/2023-09-05-101938_payment_method_data_in_payment_methods/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE payment_methods RENAME COLUMN card_details to payment_method_data;
```

Comment:
rename a column is not a backward compatibility approach. Add a new and use that.

Reply:
Is card_details added in this PR only means, remove the card detail migration itself.